lib/annotate/annotate_models.rb

```diff
@@ -290,11 +290,14 @@ def get_schema_info(klass, header, options = {})
             end
           end
         end
+
+        # Generate the column name optionally including column comments
         col_name = if with_comments?(klass, options) && col.comment
```

Comment:
I considered extracting this block into a method (something like `sanitized_column_name` or `column_name_presenter`), but it didn't seem to follow current coding conventions.

Reply:
Up to you how you want to handle this. Rubocop (currently) complains a lot about the project. We can always add a skip for it while it's being refactored.

Reply:
#791 should have resolved this